external_services/templates/external_services/lti_form.html

```diff
@@ -17,38 +17,73 @@
 		to the provider of this service.
 		{% endblocktrans %}
 	</p>
+	{% if service.enable_api_access %}
+	<p>
+		{% blocktrans %}
+		For this service also API token is transferred.
+		Thus this service will have full access to A+ API with your privileges and
+		could possibly do harm in you name.
```

Comment:
This is all true, in theory. However, as in practice many of these services are not truly external but maintained by the same people who run the course, I would like either to be able to edit this text as a user (course owner),  or use softer tone in the phrasing. How about "could possible act as you". Another question is whether students will benefit from "System administrator can disable this feature for this service" - as I would assume there was a reason why the admins enabled it in the first place.